config/backends.go

```diff
@@ -64,15 +65,21 @@ func (cfg *Aerospike) validateAndLog() error {
 	if cfg.Port <= 0 {
 		return fmt.Errorf("Cannot connect to Aerospike host at port %d", cfg.Port)
 	}
-	if cfg.DefaultTTL > 0 {
-		log.Infof("config.backend.aerospike.default_ttl_seconds: %d. Note that this configuration option is being deprecated in favor of config.request_limits.max_ttl_seconds", cfg.DefaultTTL)
-	}
+
 	log.Infof("config.backend.aerospike.host: %s", cfg.Host)
 	log.Infof("config.backend.aerospike.hosts: %v", cfg.Hosts)
 	log.Infof("config.backend.aerospike.port: %d", cfg.Port)
 	log.Infof("config.backend.aerospike.namespace: %s", cfg.Namespace)
 	log.Infof("config.backend.aerospike.user: %s", cfg.User)
 
+	if cfg.DefaultTTL > 0 {
+		log.Infof("config.backend.aerospike.default_ttl_seconds: %d. Note that this configuration option is being deprecated in favor of config.request_limits.max_ttl_seconds", cfg.DefaultTTL)
+	}
+
+	if cfg.ConnectionIdleTimeout > 0 {
+		log.Infof("config.backend.aerospike.connection_idle_timeout: %d. Will substitute Aerospike's default 55 seconds.", cfg.ConnectionIdleTimeout)
```

Comment:
I believe we can skip this part of the log message: `Will substitute Aerospike's default 55 seconds.` as when someone's overriding that value, the value they have set is what we should log. Logging the default value that it will substitute won't help much IMHO.

Reply:
Corrected